Enyim.Caching/Configuration/ConfigurationHelper.cs

```diff
@@ -98,9 +98,8 @@ public static EndPoint ResolveToEndPoint(string host, int port)
             // parse as an IP address
             if (!IPAddress.TryParse(host, out address))
             {
-                //Call System.Net.Dns.GetHostEntry(string hostNameOrAddress) that implements for netstandard 2.0
-                var method = typeof(System.Net.Dns).GetMethod("GetHostEntry", BindingFlags.Public | BindingFlags.Static);
-                var addresses = ((IPHostEntry)method.Invoke(null, new object[] { host })).AddressList;
+                var hostEntry = Dns.GetHostEntryAsync(host).Result;
```

Comment:
Okay, it appear the other method that was being used is no longer a part of the DNS class.  Do you have any other suggestions?